components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/MutualSSLManager.java

```diff
@@ -225,28 +226,35 @@ protected static String getPropertyValue(String key) {
      */
     private static void resolveSecrets(Properties properties) {
 
-        String protectedTokens = (String) properties.get(PROTECTED_TOKENS);
-
-        if (StringUtils.isNotBlank(protectedTokens)) {
-            String secretProvider = (String) properties.get(SECRET_PROVIDER);
-            SecretResolver secretResolver;
-
-            if (StringUtils.isBlank(secretProvider)) {
-                properties.put(SECRET_PROVIDER, DEFAULT_CALLBACK_HANDLER);
+        String secretProvider = (String) properties.get(SECRET_PROVIDER);
+        if (StringUtils.isBlank(secretProvider)) {
+            properties.put(SECRET_PROVIDER, DEFAULT_CALLBACK_HANDLER);
+        }
+        SecretResolver secretResolver = SecretResolverFactory.create(properties);
+        if (secretResolver != null && secretResolver.isInitialized()) {
+            for (Map.Entry<Object, Object> entry : properties.entrySet()) {
+                String key = entry.getKey().toString();
+                String value = entry.getValue().toString();
+                if (value != null) {
+                    value = MiscellaneousUtil.resolve(value, secretResolver);
+                }
+                properties.put(key, value);
             }
-
-            secretResolver = SecretResolverFactory.create(properties, "");
+        }
+        // Support the protectedToken alias used for encryption. ProtectedToken alias is deprecated
```

Comment:
Fix comment formatting issues in all instances.

Reply:
Fixed - https://github.com/wso2/carbon-identity-framework/pull/2621